caravel/bin/caravel

```diff
@@ -36,8 +37,9 @@ manager.add_command('db', MigrateCommand)
     help="Specify the timeout (seconds) for the gunicorn web server")
 def runserver(debug, port, timeout, workers):
     """Starts a Caravel web server"""
-    debug = debug or config.get("DEBUG")
-    if debug:
+    debug = config.get('LOG_LEVEL')
```

Comment:
I see. I noticed that but was wondering what if debug = True and Log Level = Debug.  I have reverted back the setting for running web server as debug mode true/false